feature-versions-impl/src/main/java/io/novafoundation/nova/feature_versions_impl/data/LocalAppVersionProvider.kt

```diff
@@ -19,6 +19,6 @@ class PackageManagerAppVersionProvider(
         } else {
             context.packageManager.getPackageInfo(context.packageName, 0)
         }
-        return packageInfo.versionName
+        return packageInfo.versionName!!
```

Comment:
It seems like we have two completely dublicate `AppVersionProvider`. Maybe we can remove one of them?